tests/test_helpers.py

```
@@ -541,6 +546,55 @@ def test_parent_makefile_test_target_uses_requisite_pytest_command() -> None:
+    capture = tmp_path / "uvx-args.txt"
+    shim = tmp_path / "uvx"
+    shim.write_text(
+        f"#!/bin/sh\nprintf '%s\\n' \"$@\" > {shlex.quote(str(capture))}\n",
+        encoding="utf-8",
+    )
+    shim.chmod(0o755)
+    environment = os.environ.copy()
+    environment["PATH"] = f"{tmp_path}{os.pathsep}{environment['PATH']}"
+
+    result = subprocess.run(
+        ["make", "typecheck"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)